extraction/extractor.py

```diff
@@ -6,6 +6,7 @@
 
 import requests
 from bs4 import BeautifulSoup, Comment
+from requests.exceptions import InvalidSchema, MissingSchema
```

Comment:
extraction/extractor.py:11:0: E0611: No name 'NoneTypeMarkupError' in module 'error' (no-name-in-module)
extraction/extractor.py:11:0: E0611: No name 'WebsiteConnectionError' in module 'error' (no-name-in-module)
extraction/extractor.py:11:0: E0611: No name 'WebsiteNotStampifiableError' in module 'error' (no-name-in-module)

Reply:
Works fine for me :/
$ pylint extraction/

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

Reply:
Did you change your pylintrc file?

Reply:
No